### ISSUE TITLE
Delete website is a dangerous function for files

### ISSUE THREAD

**Issue by AmiOta** [litespeedtech/cyberpanel#452]
Hi,

The function that deletes a website also removes the files from the Linux user in the following lines.

This is extremely dangerous if you do not have a backup policy on the servers when a malicious or accidental deletion is carried out, also if it is combined with the client bug where the SSH user is duplicated (https://github.com/usmannasir/cyberpanel/issues/450), it results in the elimination of multiple legitimate sites in the server (was what happened to me on a website with thousands of daily visits, fortunately, I manage a backup policy).

https://github.com/usmannasir/cyberpanel/blob/013da4e4960f8192051c732e21c18094bba03ce1/plogical/vhost.py#L504
https://github.com/usmannasir/cyberpanel/blob/013da4e4960f8192051c732e21c18094bba03ce1/plogical/vhost.py#L412

Happen the same with the database:

https://github.com/usmannasir/cyberpanel/blob/013da4e4960f8192051c732e21c18094bba03ce1/plogical/vhost.py#L393

In conclusion, the policy to remove any website should be less rigid, simply unlinking the references in the cyberpanel database would be enough, keeping the files and databases of the site, since they are the greatest asset of any company and an accidental removal could be a total disaster.

Thx.

**Comment by usmannasir** [comment]
Deleting the website is very clear that everything will be deleted. Right now when you delete the child-domain only its reference from database is removed but files and databases remain there and people are already not happy about this. 

I will not consider this as issue, as delete means final removal of site.
